test/box_test.dart

```diff
@@ -96,13 +96,13 @@ void main() {
     });
 
     test('Box.close', () async {
-      await collection.close();
+      await collection.closeDb();
     });
 
     test(
       'Box.delete',
       () async {
-        await BoxCollection.delete(
+        await collection.deleteDb(
```

Comment:
NIT: test name doesn't match the function name anymore.

Reply:
cleaned up